ci/properties/webpack.properties.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Webpack",
+    "description": "Build a Webpack project with npm.",
+    "iconName": "nodejs",
+    "categories": ["JavaScript", "npm", "Webpack"]
```

Comment:
Shouldn't we change `javascript` to `node`? Should node be added as a language to Linguist?

Reply:
That would be a redundancy as `scout` already detects `npm` and `yarn`. Language gets higher priority than non-language stacks Keeping `JavaScript` as it is.

Reply:
Existing node [template](https://github.com/actions/starter-workflows/blob/main/ci/properties/node.js.properties.json) specifies both "Javascript" and "node". How should we rationalize these two?
In other words, javascript is the language and node is the runtime. Should we treat them differently or as you say npm/yarn is a proxy for node runtime?

Reply:
"Node" in the existing template really has no mapping with any property of the repo. Node can be consumed in multiple ways based on the packagemanager, and the workflow changes as per the packagamanager used. Thus I feel proxying node runtime with npm/yarn should be the way to go

Reply:
Also adding "TyepScript" as a dependency, as these all can be used with ts - https://www.typescriptlang.org/docs/handbook/integrating-with-build-tools.html